app/src/main/java/com/uc/mymovies/MainActivity.kt

```diff
@@ -77,9 +80,9 @@ class MainActivity : AppCompatActivity() {
                 hideSoftKeyboard(searchFilter);
         }
 
-        searchButton.setOnClickListener({
-            hideSoftKeyboard(searchFilter);
-        });
+        searchButton.setOnClickListener {
```

Comment:
Removed unnecessary parenthesis.

Reply:
This appeared in a previous review as well.